test/index.ts

```diff
@@ -289,15 +289,33 @@ describe('user list query', () => {
     const token = sign({ email: loginUser.email }, jwtTokenSecret, { expiresIn: '1d' });
     const query = await userListQuery(url, token, quantity);
 
-    expect(quantity).to.be.lte(query.data.data.users.length);
-    expect(query.data.data.users[0]).to.have.all.keys(['name', 'id', 'birthDate', 'email']);
+    expect(query.data.data.data.users[0]).to.have.all.keys(['name', 'id', 'birthDate', 'email']);
+    expect(quantity).to.be.lte(query.data.data.data.users.length);
   });
 
   it('default quantity parameter equal 10', async () => {
     const defaultQuantity = 10;
     const token = sign({ email: loginUser.email }, jwtTokenSecret, { expiresIn: '1d' });
     const query = await userListQuery(url, token);
 
-    expect(defaultQuantity).to.be.eq(query.data.data.users.length);
+    expect(defaultQuantity).to.be.eq(query.data.data.data.users.length);
+  });
+
+  it('previousPage should returns false and true for nextPage', async () => {
```

Comment:
aqui é um exemplo do que comentei no PR anterior, poderia ter uma descrição como `should return previousPage as false and nextPage as true when ....`
é importante ter a descrição "when" no it até para quem faz o CR poder validar se você realmente montou o cenário certinho no it